src/tests/cmocka/test_pam_srv.c

```diff
@@ -4021,13 +4123,23 @@ int main(int argc, const char *argv[])
                                         pam_test_setup, pam_test_teardown),
         cmocka_unit_test_setup_teardown(test_pam_preauth_expired_crl_file_soft,
                                         pam_test_setup, pam_test_teardown),
+        cmocka_unit_test_setup_teardown(test_pam_preauth_crl_valid_crl_invalid_files,
+                                        pam_test_setup, pam_test_teardown),
 #endif /* HAVE_FAKETIME */
         cmocka_unit_test_setup_teardown(test_pam_preauth_ocsp,
                                         pam_test_setup, pam_test_teardown),
         cmocka_unit_test_setup_teardown(test_pam_preauth_ocsp_no_ocsp,
                                         pam_test_setup, pam_test_teardown),
         cmocka_unit_test_setup_teardown(test_pam_preauth_ocsp_soft_ocsp,
                                         pam_test_setup, pam_test_teardown),
+        cmocka_unit_test_setup_teardown(test_pam_preauth_first_crl_another_ca_files,
+                                        pam_test_setup, pam_test_teardown),
+        cmocka_unit_test_setup_teardown(test_pam_preauth_last_crl_another_ca_files,
+                                        pam_test_setup, pam_test_teardown),
+        cmocka_unit_test_setup_teardown(test_pam_preauth_crl_another_ca_crl_invalid_files,
+                                        pam_test_setup, pam_test_teardown),
+        cmocka_unit_test_setup_teardown(test_pam_preauth_crl_invalid_crl_another_ca_files,
+                                        pam_test_setup, pam_test_teardown),
```

Comment:
The last two tests are using the expired CRL as well, so I guess they should be moved to the `#if HAVE_FAKETIME` block too.

Reply:
You are right. Thanks for catching that.